addons/arsenal/functions/fnc_fillLoadoutsList.sqf

```diff
@@ -104,6 +104,11 @@ if (GVAR(currentLoadoutsTab) != IDC_buttonSharedLoadouts) then {
         } else {
             if (_unavailableItemsList isNotEqualTo []) then {
                 _contentPanelCtrl lnbSetColor [[_newRow, 1], [1, 1, 1, 0.25]]; // Gray
+                if (GVAR(unavailableItemsTooltip)) then {
+                    private _unavailableItemsUnique = [];
+                    {_unavailableItemsUnique pushBackUnique _x} forEach _unavailableItemsList;
```

Comment:
arrayIntersect may be faster

Reply:
Slower at higher array counts IIRC

Reply:
Would make sense, might test later if I remember

Reply:
Fair enough, done

Reply:
Looping seemed to always be slower. These were all tested with arrays of unique numbers since that was the easiest thing to create.
```sqf
pushBackUnique
  10 Elements - Execution Time: 0.0030 ms  |  Cycles: 10000/10000  |  Total Time: 30 ms
 100 Elements - Execution Time: 0.0640 ms  |  Cycles: 10000/10000  |  Total Time: 640 ms
1000 Elements - Execution Time: 5.2507 ms  |  Cycles: 191/10000    |  Total Time: 1003 ms

arrayIntersect
  10 Elements - Execution Time: 0.0015 ms  |  Cycles: 10000/10000  |  Total Time: 15 ms
 100 Elements - Execution Time: 0.0544 ms  |  Cycles: 10000/10000  |  Total Time: 544 ms
1000 Elements - Execution Time: 5.1586 ms  |  Cycles: 194/10000    |  Total Time: 1001 ms
```